packages/browser-pyright/webpack.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = (_, { mode }) => {
             devtoolModuleFilenameTemplate:
                 mode === 'development' ? '../[resource-path]' : monorepoResourceNameMapper('pyright'),
             clean: true,
+            chunkLoading: 'import-scripts',
```

Comment:
Needed for web worker chunk loading (otherwise it tries to add a script tag which doesn't end well).